src/components/ClaimableBalances.tsx

```diff
@@ -70,7 +77,7 @@ export const ClaimableBalances = () => {
   return (
     <div className="ClaimableBalances DataSection">
       <Layout.Inset>
-        <Heading2> Claimable Balance(s)</Heading2>
+        <Heading2> {getClaimBalanceHeader()}</Heading2>
```

Comment:
Extra space at the beginning here. Sorry for being so picky about these things. :blush:

Reply:
Good catchn thank you!